hack/generator/pkg/consts/generation_consts.go

```diff
@@ -0,0 +1,7 @@
+package consts
+
+const (
+	// Note that this format is actually an official specification in go:  https://github.com/golang/go/issues/13560
+	CodeGenerationComment = "Code generated by k8s-infra-gen. DO NOT EDIT."
+	CodeGeneratedFilePrefix = "zz_generated." // TODO: Technically this is not required now, could remove it?
```

Comment:
Do we like the zz_generated prefix or no?

Reply:
We don't actually need any changes to the file naming (I had originally written things so that we did and then realized that go had an official "how to detect generated files" which is based on a comment in the file so I moved to use that.

I can just revert the `zz_generated` and leave things as is. We should correctly detect what files are generated by us vs other tools/handcrafted and it should in theory all play nicely together without any specific file naming convention at all.

Reply:
I do think there's value in having generated files detectable by filename - it's another clue to nudge future devs who might otherwise manually change the files.

I'm partial to `_gen.go` - it seems more in line with the way Go source files are named than `.gen.go`.

Reply:
Pushed an updated to move to `_gen.go` suffix